EIPS/eip-crosschain-id.md

```diff
@@ -0,0 +1,131 @@
+---
+eip: <to be assigned>
+title: Crosschain Identifier Specification
+author: Weijia Zhang <weijia@wanchain.org>, Peter Robinson <peter.robinson@consensys.net>
+discussions-to:
+status: Draft
+type: Standards Track
+category: ERC
```

Comment:
If you're proposing a standard for signing messages that is not compatible with EIP-155, the category should be `Core`.

Reply:
Yes, changed the document.